src/pages/media_uploads/media-upload-list-page.js

```diff
@@ -200,7 +201,7 @@ const mapStateToProps = ({
   mediaUploadListState
 }) => ({
   currentSummit: currentSummitState.currentSummit,
-  allSummits: directoryState.allSummits,
+  allSummits: directoryState.summits,
```

Comment:
this should be removed and  please use 
https://github.com/fntechgit/summit-admin/commit/5340e7dc5e1b0bb6ce91a7c3359e1bb4f1109a47#diff-b28fd20f5d203a42ef103dea9165e2fe73987d15bcce7e16c2028913a1cf9c07